index.html

```diff
@@ -55,118 +64,128 @@ <h2 class="contactTitle"><a href="#">LET’S CONNECT</a></h2>
-            <li>Back End Dev</li>
-            <li><img src="assets/Counter.png" alt="" /></li>
-            <li>2015</li>
-          </ul>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <ul class="work-lang">
-            <li>HTML</li>
-            <li>CSS</li>
-            <li>JavaScript</li>
-          </ul>
-          <button>See project</button>
+          <div class="card-info">
+            <h3>Multi-Post Stories</h3>
+            <ul class="skill-info">
+              <li>CANOPY</li>
+              <li><img src="assets/Counter.png" alt="" /></li>
+              <li>Back End Dev</li>
+              <li><img src="assets/Counter.png" alt="" /></li>
+              <li>2015</li>
+            </ul>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="work-lang">
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+            </ul>
+            <button>See project</button>
+          </div>
         </article>
       </section>
 
       <section class="about-section">
-        <article>
-          <h2>About Myself</h2>
-          <p>
-            Hello I’m a software developer! I can help you build a product ,
-            feature or website. Look through some of my work and experience! If
-            you like what you see and have a project you need coded, don’t
-            hesitate to contact me.
-          </p>
-          <h2 class="contactTitle"><a href="#">LET’S CONNECT</a></h2>
-          <ul class="lets-connect">
-            <li>
-              <a href="#"><img src="assets/Vector.svg" alt="" /></a>
-            </li>
-            <li>
-              <a href="#"><img src="assets/Linkedin icon-2.svg" alt="" /></a>
-            </li>
-            <li>
-              <a href="#"><img src="assets/Vector-2.svg" alt="" /></a>
-            </li>
-            <li>
-              <a href="#"><img src="assets/Vector-3.svg" alt="" /></a>
-            </li>
-            <li>
-              <a href="#"><img src="assets/Vector-4.svg" alt="" /></a>
-            </li>
-          </ul>
-          <button class="get-resume-btn">Get my resume</button>
+        <article class="about-article">
+          <div class="about-myself">
+            <h2>About Myself</h2>
+            <p>
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website. Look through some of my work and experience!
+              If you like what you see and have a project you need coded, don’t
+              hesitate to contact me.
+            </p>
+            <h2 class="contactTitle"><a href="#">LET’S CONNECT</a></h2>
+            <ul class="lets-connect">
+              <li>
+                <a href="#"><img src="assets/Vector.svg" alt="" /></a>
```

Comment:
- [ ] Please, ensure that your `img` tag always has an `alt` attribute filled, this is good for accessibility reasons, for example, for a visually impaired individual or a slow internet user. 🙂